ui/components/editor-submit-dialog.vue

```diff
@@ -5,56 +5,123 @@
     :shown="submit.state !== `closed`"
     :title="title">
 
+    <a
+      ref="downloadAnchorElement"
+      :href="downloadFileUrl"
+      :download="downloadFileName"
+      hidden />
+
+    <div v-if="submit.state === `validating`">Validating…</div>
+
     <div v-if="submit.state === `loading`">Uploading…</div>
 
+    <div v-else-if="submit.state === `ready`">
+      <template v-if="validationErrors.length || validationWarnings.length">
+        The fixture validation returned some remarks:
+
+        <ul>
+          <li v-for="message in validationErrors" :key="message">{{ message }}</li>
+          <li v-for="message in validationWarnings" :key="message">{{ message }}</li>
+        </ul>
+
+        You can try to resolve them or continue anyway with submitting your fixture
+        to the Open Fixture Library project. We will then fix those issues.
+      </template>
+      <template v-else>
+        The fixture validation was successful. You can now submit your fixture to
+        the Open Fixture Library project or only download it for private use.
```

Comment:
This might have the effect that both options seem exclusive, i.e. if I submit the fixture, I can't download it (or the other way round).

Maybe better:
> You can now submit your fixture to the Open Fixture Library project, where it will be reviewed and added to the library. Even now, you can download your current fixture for private use.